engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -364,8 +249,8 @@ object WorkflowDescriptor {
   private val writeDisabled = disabledMessage("Write to Cache", "WILL NOT be cached")
   private val readDisabled = disabledMessage("Read from Cache", "WILL ALL be executed")
 
-  private def configCallCaching(conf: Config) = lookupBooleanWithDefault(conf, "call-caching", "enabled", DefaultCallCachingValue)
-  private def lookupDockerHash(conf: Config) = lookupBooleanWithDefault(conf, "call-caching", "lookup-docker-hash", DefaultLookupDockerHash)
+  def configCallCaching(conf: Config) = lookupBooleanWithDefault(conf, "call-caching", "enabled", DefaultCallCachingValue)
+  def lookupDockerHash(conf: Config) = lookupBooleanWithDefault(conf, "call-caching", "lookup-docker-hash", DefaultLookupDockerHash)
```

Comment:
I am thinking to move this methods to the `MaterializeWorkflowDescriptorActor` (where they are being actually called). Please let me know if anyone thinks otherwise.